pkg/rtc/mediatrack.go

```diff
@@ -225,7 +225,7 @@ func (t *MediaTrack) AddReceiver(receiver *webrtc.RTPReceiver, track *webrtc.Tra
 		)
 		newWR.SetRTCPCh(t.params.RTCPChan)
 		newWR.OnCloseHandler(func() {
```

Comment:
There is a window here where `AddSubcriber` could get through. Need to think about ways to plug that window.

Reply:
Maybe, I should call `InitiateClose` from here too. Will do that. That will make this better.

Reply:
But, it brings up a different question. @cnderrauber Should this be removing all subscribers or should it be removing only subscribers for that mime? Looks like we need to remove only subscribers for that mime and let the other receivers run?

Reply:
@boks1971 We don't have api for user to unpublish one codec of track, so RemoveAllSubscriber for all receivers is ok. But mime only is ok too.

Reply:
I prefer remove them all here since it is more simple  :)

Reply:
Agreed, we do not have remove of separate, but just wanted to keep this operation receiver based :-) `ClearReceiver` will remove all subscribers for that `mime`. Eventually, all subscribers for all mimes will be removed.